Tasks/PerformActionOnMachineGroup/task.json

```diff
@@ -69,22 +69,31 @@
            "visibleRule": "Action = Block"
         },
         {
-           "name": "TimeInHours",
-           "type": "pickList",
-           "label": "Time in Hours",
-           "defaultValue": "0",
-           "required": "true",
-           "helpMarkDown": "The number of hours that the machines will be blocked for. Default value 0 indicates that the machines will be blocked indefinitely.",
-           "options": {
+            "name": "TimeInHours",
+            "type": "pickList",
+            "label": "Time in Hours",
+            "defaultValue": "0",
+            "required": "true",
+            "helpMarkDown": "The number of hours that the machines will be blocked for. Default value 0 indicates that the machines will be blocked indefinitely.",
+            "options": {
                 "0": "0",
                 "8": "8",
                 "24": "24",
                 "48": "48"
             },
-           "visibleRule": "Action = Block",
-           "properties": {
-                "EditableOptions": "True" 
+            "visibleRule": "Action = Block",
+            "properties": {
+                "EditableOptions": "True"
             }
+        },
+        {
+           "name": "WaitTimeInMinutes",
+           "type": "string",
```

Comment:
Should this be string type? How can we bind this to INT? What is max value supported?